src/cmd/troubleshoot/image.go

```diff
@@ -192,83 +196,73 @@ func checkCustomModuleImagePullable(httpClient *http.Client, componentName strin
 }
 
 func imageAvailable(httpClient *http.Client, imageUrl string, apiToken string) error {
-	if statusCode, err := connectToDockerRegistry(httpClient, "HEAD", imageUrl, "Basic", apiToken); err != nil {
+	statusCode, err := connectToDockerRegistry(httpClient, imageUrl, apiToken)
+
+	if err != nil {
 		return fmt.Errorf("registry unreachable: %w", err)
-	} else {
-		if statusCode != http.StatusOK {
-			return fmt.Errorf("image not found (status code = %d)", statusCode)
-		}
+	} else if statusCode != http.StatusOK {
+		return fmt.Errorf("image not found (status code = %d)", statusCode)
```

Comment:
should these `fmt.Errorf` calls be replaced with `errors.Wrap` since we are replacing them in other parts of the code?

Reply:
changed